<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="silius" errors="0" failures="66" skipped="13" tests="486" time="261.137" timestamp="2025-05-18T15:14:52.044930+00:00" hostname="pkrvmf6wy0o8zjz"><testcase classname="" name="tests.single.gas.test_pre_verification_gas_calculation" time="0.000"><skipped message="collection skipped">('/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/gas/test_pre_verification_gas_calculation.py', 15, 'Skipped: Slow and requires fixing')</skipped></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[only_priority_fee_bump]" time="0.255" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[only_max_fee_bump]" time="0.180" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[with_same_fee]" time="0.159" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[with_less_fee]" time="0.161" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[fee_bump_below_threshold]" time="0.157" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[fee_bump_at_threshold]" time="0.266" /><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[fee_bump_above_threshold]" time="0.268" /><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-010-banned-entity-not-allowed-banned-sender]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-010-banned-entity-not-allowed-banned-paymaster]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-010-banned-entity-not-allowed-banned-factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-020-throttled-entity-allowed-limited-count-throttled-sender]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-020-throttled-entity-allowed-limited-count-throttled-paymaster]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[GREP-020-throttled-entity-allowed-limited-count-throttled-factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[UREP-010 UREP-020-unstaked-entity-allowed-function-unstaked-sender]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[UREP-010 UREP-020-unstaked-entity-allowed-function-unstaked-paymaster]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[UREP-010 UREP-020-unstaked-entity-allowed-function-unstaked-factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_bundle.py:123: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_max_allowed_ops_unstaked_sender" time="0.769" /><testcase classname="tests.single.bundle.test_bundle" name="test_max_allowed_ops_staked_sender" time="0.896" /><testcase classname="tests.single.bundle.test_bundle" name="test_ban_user_op_access_other_ops_sender_in_bundle" time="0.783" /><testcase classname="tests.single.bundle.test_bundle" name="test_ban_user_sender_double_role_in_bundle" time="0.397" /><testcase classname="tests.single.bundle.test_bundle" name="test_stake_check_in_bundler" time="0.536" /><testcase classname="tests.single.bundle.test_codehash" name="test_codehash_changed" time="0.229"><skipped type="pytest.skip" message="no self destruct. can't check code change..">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/bundle/test_codehash.py:54: no self destruct. can't check code change..</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_paymaster" name="test_paymaster_deposit" time="0.782"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xa6543cfda365bd7ae6b7d2456cf1026ea64af81c9bdffa6fe1ed189272cd5f39', id=98)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_paymaster.py:48: in test_paymaster_deposit
    assert_rpc_error(res, "too low", RPCErrorCode.PAYMASTER_DEPOSIT_TOO_LOW)
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xa6543cfda365bd7ae6b7d2456cf1026ea64af81c9bdffa6fe1ed189272cd5f39', id=98)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][paymaster][no_storage][ok-]" time="0.372" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][paymaster][no_storage][ok-CALL:&gt;]" time="0.586" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][paymaster][no_storage][ok-DELEGATECALL:&gt;]" time="0.673" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_read][drop-]" time="0.385" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_read][drop-CALL:&gt;]" time="0.611" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_read][drop-DELEGATECALL:&gt;]" time="0.707" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_write][drop-]" time="0.398" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_write][drop-CALL:&gt;]" time="0.623" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage_write][drop-DELEGATECALL:&gt;]" time="0.812" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tstore][drop-]" time="0.520"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tstore][drop-CALL:&gt;]" time="0.766"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tstore][drop-DELEGATECALL:&gt;]" time="0.853"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tload][drop-]" time="0.514"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tload][drop-CALL:&gt;]" time="0.759"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-031)]unstaked][paymaster][transient_storage_tload][drop-DELEGATECALL:&gt;]" time="0.846"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:21: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage][drop-]" time="0.535" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage][drop-CALL:&gt;]" time="0.797" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage][drop-DELEGATECALL:&gt;]" time="0.873" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage_struct][drop-]" time="0.696" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage_struct][drop-CALL:&gt;]" time="0.882" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][paymaster_reference_storage_struct][drop-DELEGATECALL:&gt;]" time="0.958" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][paymaster][account_storage][ok-]" time="0.527" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][paymaster][account_storage][ok-CALL:&gt;]" time="0.654" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][paymaster][account_storage][ok-DELEGATECALL:&gt;]" time="0.840" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-010)]unstaked][paymaster][account_transient_storage][ok-]" time="0.540" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-010)]unstaked][paymaster][account_transient_storage][ok-CALL:&gt;]" time="0.787" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-070(STO-010)]unstaked][paymaster][account_transient_storage][ok-DELEGATECALL:&gt;]" time="0.881" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage][ok-]" time="0.521" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage][ok-CALL:&gt;]" time="0.770" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.855" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage_struct][ok-]" time="0.621" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage_struct][ok-CALL:&gt;]" time="0.868" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="1.043" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][drop-]" time="1.147" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][drop-CALL:&gt;]" time="1.010" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][drop-DELEGATECALL:&gt;]" time="1.200" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050]unstaked][paymaster][context][drop-]" time="0.232" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050]unstaked][paymaster][context][drop-CALL:&gt;]" time="0.259" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050]unstaked][paymaster][context][drop-DELEGATECALL:&gt;]" time="0.296" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][external_storage_read][drop-]" time="0.482" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][external_storage_read][drop-CALL:&gt;]" time="0.718" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][external_storage_read][drop-DELEGATECALL:&gt;]" time="0.912" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][out_of_gas][drop-]" time="0.474" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][out_of_gas][drop-CALL:&gt;]" time="0.786" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.878" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][sstore_out_of_gas][drop-]" time="0.563" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][sstore_out_of_gas][drop-CALL:&gt;]" time="0.797" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.781" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][paymaster][no_storage][ok-]" time="0.490" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][paymaster][no_storage][ok-CALL:&gt;]" time="0.709" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][paymaster][no_storage][ok-DELEGATECALL:&gt;]" time="0.821" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_read][ok-]" time="0.406" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_read][ok-CALL:&gt;]" time="0.633" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_read][ok-DELEGATECALL:&gt;]" time="0.816" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_write][ok-]" time="0.519" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_write][ok-CALL:&gt;]" time="0.636" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage_write][ok-DELEGATECALL:&gt;]" time="0.832" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage][ok-]" time="0.659" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage][ok-CALL:&gt;]" time="0.923" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage][ok-DELEGATECALL:&gt;]" time="0.999" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage_struct][ok-]" time="0.618" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage_struct][ok-CALL:&gt;]" time="0.998" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][paymaster_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="1.087" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][paymaster_reference_storage_struct][ok-]" time="0.627" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][paymaster_reference_storage_struct][ok-CALL:&gt;]" time="1.016" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][paymaster_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.976" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][paymaster][account_storage][ok-]" time="0.448" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][paymaster][account_storage][ok-CALL:&gt;]" time="0.801" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][paymaster][account_storage][ok-DELEGATECALL:&gt;]" time="0.794" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage][ok-]" time="0.572" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage][ok-CALL:&gt;]" time="0.916" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.989" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage_struct][ok-]" time="0.713" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage_struct][ok-CALL:&gt;]" time="0.898" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="1.078" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]staked][paymaster][account_reference_storage_init_code][ok-]" time="0.789" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]staked][paymaster][account_reference_storage_init_code][ok-CALL:&gt;]" time="1.158" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]staked][paymaster][account_reference_storage_init_code][ok-DELEGATECALL:&gt;]" time="1.117" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050]staked][paymaster][context][ok-]" time="0.362" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050]staked][paymaster][context][ok-CALL:&gt;]" time="0.386" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050]staked][paymaster][context][ok-DELEGATECALL:&gt;]" time="0.526" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_write][drop-]" time="0.532" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_write][drop-CALL:&gt;]" time="0.765" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_write][drop-DELEGATECALL:&gt;]" time="0.856" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_read][ok-]" time="0.508" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_read][ok-CALL:&gt;]" time="0.755" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_read][ok-DELEGATECALL:&gt;]" time="0.935" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][out_of_gas][drop-]" time="0.602" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][out_of_gas][drop-CALL:&gt;]" time="0.818" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.809" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][sstore_out_of_gas][drop-]" time="0.586" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][sstore_out_of_gas][drop-CALL:&gt;]" time="0.722" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.907" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][no_storage][ok-]" time="0.314" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][no_storage][ok-CALL:&gt;]" time="0.482" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][no_storage][ok-DELEGATECALL:&gt;]" time="0.600" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_read][drop-]" time="0.419" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_read][drop-CALL:&gt;]" time="0.695" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_read][drop-DELEGATECALL:&gt;]" time="0.509" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_write][drop-]" time="0.422" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_write][drop-CALL:&gt;]" time="0.493" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage_write][drop-DELEGATECALL:&gt;]" time="0.515" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][factory_reference_storage][drop-]" time="0.442" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][factory_reference_storage][drop-CALL:&gt;]" time="0.643" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][factory_reference_storage][drop-DELEGATECALL:&gt;]" time="0.734" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][factory][factory_reference_storage_struct][drop-]" time="0.343" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][factory][factory_reference_storage_struct][drop-CALL:&gt;]" time="0.582" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][factory][factory_reference_storage_struct][drop-DELEGATECALL:&gt;]" time="0.649" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][factory][account_storage][ok-]" time="0.325" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][factory][account_storage][ok-CALL:&gt;]" time="0.605" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][factory][account_storage][ok-DELEGATECALL:&gt;]" time="0.620" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage][drop-]" time="0.330" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage][drop-CALL:&gt;]" time="0.630" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage][drop-DELEGATECALL:&gt;]" time="0.698" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage_struct][drop-]" time="0.437" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage_struct][drop-CALL:&gt;]" time="0.680" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage_struct][drop-DELEGATECALL:&gt;]" time="0.740" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][external_storage_read][drop-]" time="0.344" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][external_storage_read][drop-CALL:&gt;]" time="0.638" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][external_storage_read][drop-DELEGATECALL:&gt;]" time="0.573" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-042]unstaked][factory][EXTCODEx_CALLx_undeployed_sender][ok-]" time="0.351" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-042]unstaked][factory][EXTCODEx_CALLx_undeployed_sender][ok-CALL:&gt;]" time="0.472" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-042]unstaked][factory][EXTCODEx_CALLx_undeployed_sender][ok-DELEGATECALL:&gt;]" time="0.620" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODESIZE_undeployed_contract][drop-]" time="0.396" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODESIZE_undeployed_contract][drop-CALL:&gt;]" time="0.541" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODESIZE_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.712" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODEHASH_undeployed_contract][drop-]" time="0.302" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODEHASH_undeployed_contract][drop-CALL:&gt;]" time="0.640" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODEHASH_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.710" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODECOPY_undeployed_contract][drop-]" time="0.404" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODECOPY_undeployed_contract][drop-CALL:&gt;]" time="0.646" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODECOPY_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.702" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALL_undeployed_contract][drop-]" time="0.290" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALL_undeployed_contract][drop-CALL:&gt;]" time="0.490" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.641" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALLCODE_undeployed_contract][drop-]" time="0.396" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALLCODE_undeployed_contract][drop-CALL:&gt;]" time="0.514" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALLCODE_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.672" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][DELEGATECALL_undeployed_contract][drop-]" time="0.304" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][DELEGATECALL_undeployed_contract][drop-CALL:&gt;]" time="0.649" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][DELEGATECALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.612" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][STATICCALL_undeployed_contract][drop-]" time="0.296" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][STATICCALL_undeployed_contract][drop-CALL:&gt;]" time="0.640" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][STATICCALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.593" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][out_of_gas][drop-]" time="0.529" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][out_of_gas][drop-CALL:&gt;]" time="0.722" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.713" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][sstore_out_of_gas][drop-]" time="0.351" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][sstore_out_of_gas][drop-CALL:&gt;]" time="0.634" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.556" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][factory][no_storage][ok-]" time="0.337" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][factory][no_storage][ok-CALL:&gt;]" time="0.603" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][factory][no_storage][ok-DELEGATECALL:&gt;]" time="0.522" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_read][ok-]" time="0.446" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_read][ok-CALL:&gt;]" time="0.610" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_read][ok-DELEGATECALL:&gt;]" time="0.634" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_write][ok-]" time="0.340" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_write][ok-CALL:&gt;]" time="0.618" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage_write][ok-DELEGATECALL:&gt;]" time="0.532" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage][ok-]" time="0.467" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage][ok-CALL:&gt;]" time="0.664" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage][ok-DELEGATECALL:&gt;]" time="0.620" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage_struct][ok-]" time="0.470" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage_struct][ok-CALL:&gt;]" time="0.728" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][factory_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.779" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][factory][account_storage][ok-]" time="0.444" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][factory][account_storage][ok-CALL:&gt;]" time="0.523" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][factory][account_storage][ok-DELEGATECALL:&gt;]" time="0.641" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage][ok-]" time="0.356" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage][ok-CALL:&gt;]" time="0.579" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.718" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage_struct][ok-]" time="0.467" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage_struct][ok-CALL:&gt;]" time="0.701" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.777" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_write][drop-]" time="0.468" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_write][drop-CALL:&gt;]" time="0.664" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_write][drop-DELEGATECALL:&gt;]" time="0.715" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_read][ok-]" time="0.362" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_read][ok-CALL:&gt;]" time="0.654" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_read][ok-DELEGATECALL:&gt;]" time="0.696" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][out_of_gas][drop-]" time="0.654" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][out_of_gas][drop-CALL:&gt;]" time="0.611" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.745" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][sstore_out_of_gas][drop-]" time="0.380" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][sstore_out_of_gas][drop-CALL:&gt;]" time="0.658" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.681" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][no_storage][ok-]" time="0.781" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][no_storage][ok-CALL:&gt;]" time="0.465" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][no_storage][ok-DELEGATECALL:&gt;]" time="0.568" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][account][account_storage][ok-]" time="0.289" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][account][account_storage][ok-CALL:&gt;]" time="0.594" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][account][account_storage][ok-DELEGATECALL:&gt;]" time="0.589" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage][ok-]" time="0.397" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage][ok-CALL:&gt;]" time="0.626" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.519" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][drop-]" time="0.402" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][drop-CALL:&gt;]" time="0.426" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][drop-DELEGATECALL:&gt;]" time="0.504" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][ok-]" time="0.936"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=496)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=496)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][ok-CALL:&gt;]" time="0.672"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=498)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=498)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------
CALL:&gt; funding test_rules_target 0x746765783473878625bCb1a58575E595e6029aeC

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][ok-DELEGATECALL:&gt;]" time="0.765"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=500)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=500)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][ok-]" time="0.888"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=502)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=502)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][ok-CALL:&gt;]" time="1.152"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=504)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=504)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][ok-DELEGATECALL:&gt;]" time="1.240"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=506)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:864: in test_rule
    case.assert_func(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=506)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage_struct][ok-]" time="0.298" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage_struct][ok-CALL:&gt;]" time="0.637" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.641" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][external_storage_read][drop-]" time="0.409" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][external_storage_read][drop-CALL:&gt;]" time="0.621" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][external_storage_read][drop-DELEGATECALL:&gt;]" time="0.618" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][out_of_gas][drop-]" time="0.396" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][out_of_gas][drop-CALL:&gt;]" time="0.676" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.676" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][sstore_out_of_gas][drop-]" time="0.325" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][sstore_out_of_gas][drop-CALL:&gt;]" time="0.625" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.518" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][account][no_storage][ok-]" time="0.405" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][account][no_storage][ok-CALL:&gt;]" time="0.594" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][account][no_storage][ok-DELEGATECALL:&gt;]" time="0.614" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][account][account_storage][ok-]" time="0.419" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][account][account_storage][ok-CALL:&gt;]" time="0.616" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][account][account_storage][ok-DELEGATECALL:&gt;]" time="0.506" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage][ok-]" time="0.426" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage][ok-CALL:&gt;]" time="0.646" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage][ok-DELEGATECALL:&gt;]" time="0.540" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage_struct][ok-]" time="0.425" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage_struct][ok-CALL:&gt;]" time="0.671" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage_struct][ok-DELEGATECALL:&gt;]" time="0.668" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][out_of_gas][drop-]" time="0.520" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][out_of_gas][drop-CALL:&gt;]" time="0.714" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][out_of_gas][drop-DELEGATECALL:&gt;]" time="0.704" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][sstore_out_of_gas][drop-]" time="0.342" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][sstore_out_of_gas][drop-CALL:&gt;]" time="0.543" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][sstore_out_of_gas][drop-DELEGATECALL:&gt;]" time="0.553" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_write][drop-]" time="0.440" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_write][drop-CALL:&gt;]" time="0.650" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_write][drop-DELEGATECALL:&gt;]" time="0.649" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_read][ok-]" time="0.329" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_read][ok-CALL:&gt;]" time="0.546" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_read][ok-DELEGATECALL:&gt;]" time="0.643" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-011]unstaked][account][entryPoint_call_balanceOf][drop-]" time="0.448" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-011]unstaked][account][entryPoint_call_balanceOf][drop-CALL:&gt;]" time="0.649" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-011]unstaked][account][entryPoint_call_balanceOf][drop-DELEGATECALL:&gt;]" time="0.644" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-061]unstaked][account][eth_value_transfer_forbidden][drop-]" time="0.317" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-061]unstaked][account][eth_value_transfer_forbidden][drop-CALL:&gt;]" time="0.646" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-061]unstaked][account][eth_value_transfer_forbidden][drop-DELEGATECALL:&gt;]" time="0.642" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-053]unstaked][account][eth_value_transfer_entryPoint][ok-]" time="0.332" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-053]unstaked][account][eth_value_transfer_entryPoint][ok-CALL:&gt;]" time="0.559" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-053]unstaked][account][eth_value_transfer_entryPoint][ok-DELEGATECALL:&gt;]" time="0.562" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-052]unstaked][account][eth_value_transfer_entryPoint_depositTo][ok-]" time="0.437" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-052]unstaked][account][eth_value_transfer_entryPoint_depositTo][ok-CALL:&gt;]" time="0.693" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-052]unstaked][account][eth_value_transfer_entryPoint_depositTo][ok-DELEGATECALL:&gt;]" time="0.588" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODESIZE_undeployed_contract][drop-]" time="0.368" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODESIZE_undeployed_contract][drop-CALL:&gt;]" time="0.601" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODESIZE_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.502" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODEHASH_undeployed_contract][drop-]" time="0.371" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODEHASH_undeployed_contract][drop-CALL:&gt;]" time="0.612" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODEHASH_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.504" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODECOPY_undeployed_contract][drop-]" time="0.266" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODECOPY_undeployed_contract][drop-CALL:&gt;]" time="0.606" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODECOPY_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.502" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODESIZE_entrypoint][drop-]" time="0.278" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODESIZE_entrypoint][drop-CALL:&gt;]" time="0.499" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODESIZE_entrypoint][drop-DELEGATECALL:&gt;]" time="0.587" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODEHASH_entrypoint][drop-]" time="0.381" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODEHASH_entrypoint][drop-CALL:&gt;]" time="0.593" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODEHASH_entrypoint][drop-DELEGATECALL:&gt;]" time="0.596" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODECOPY_entrypoint][drop-]" time="0.382" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODECOPY_entrypoint][drop-CALL:&gt;]" time="0.601" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODECOPY_entrypoint][drop-DELEGATECALL:&gt;]" time="0.627" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALL_undeployed_contract][drop-]" time="0.356" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALL_undeployed_contract][drop-CALL:&gt;]" time="0.577" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.575" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALLCODE_undeployed_contract][drop-]" time="0.258" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALLCODE_undeployed_contract][drop-CALL:&gt;]" time="0.600" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALLCODE_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.591" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][DELEGATECALL_undeployed_contract][drop-]" time="0.368" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][DELEGATECALL_undeployed_contract][drop-CALL:&gt;]" time="0.604" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][DELEGATECALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.498" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][STATICCALL_undeployed_contract][drop-]" time="0.363" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][STATICCALL_undeployed_contract][drop-CALL:&gt;]" time="0.495" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][STATICCALL_undeployed_contract][drop-DELEGATECALL:&gt;]" time="0.496" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-062]unstaked][account][CALL_undeployed_contract_allowed_precompile][ok-]" time="0.378" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-062]unstaked][account][CALL_undeployed_contract_allowed_precompile][ok-CALL:&gt;]" time="0.550" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-062]unstaked][account][CALL_undeployed_contract_allowed_precompile][ok-DELEGATECALL:&gt;]" time="0.538" /><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_eip_7702_tx" time="0.074"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='sender 0x9628…169c is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=670)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:36: in test_send_eip_7702_tx
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='sender 0x9628…169c is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=670)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_eip_7702_tx_with_initcode" time="0.071"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='Invalid params', data='invalid length 4, expected a (both 0x-prefixed or not) hex string or byte array containing 20 bytes at line 1 column 92', id=672)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:85: in test_send_eip_7702_tx_with_initcode
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='Invalid params', data='invalid length 4, expected a (both 0x-prefixed or not) hex string or byte array containing 20 bytes at line 1 column 92', id=672)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_eip_7702_tx_with_flag_no_initcode" time="0.072"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='Invalid params', data='invalid length 4, expected a (both 0x-prefixed or not) hex string or byte array containing 20 bytes at line 1 column 92', id=674)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:127: in test_send_eip_7702_tx_with_flag_no_initcode
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='Invalid params', data='invalid length 4, expected a (both 0x-prefixed or not) hex string or byte array containing 20 bytes at line 1 column 92', id=674)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_post_eip_7702_tx[0]" time="0.071"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='sender 0xb95f…4b20 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=676)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:156: in test_send_post_eip_7702_tx
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='sender 0xb95f…4b20 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=676)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_post_eip_7702_tx[1337]" time="0.071"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='sender 0x53a4…1e5f is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=678)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:156: in test_send_post_eip_7702_tx
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='sender 0x53a4…1e5f is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=678)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_bad_eip_7702_drop_userop" time="0.073"><failure message="AssertionError: assert -32602 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;&#10; +  where -32602 = Error(code=-32602, message='sender 0x9b16…671d is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=680).code">tests/single/eip7702/test_eip7702_tuple_userop.py:208: in test_send_bad_eip_7702_drop_userop
    assert_rpc_error(response, "", RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT)
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AssertionError: assert -32602 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;
E    +  where -32602 = Error(code=-32602, message='sender 0x9b16…671d is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=680).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_nonsender_eip_7702_drop_userop" time="0.161"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xa2f4134516fab4f38dfc2485820ac2b86352679f2af0e307c848adc3ceee43cf', id=682)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:223: in test_send_nonsender_eip_7702_drop_userop
    assert_rpc_error(userop.send(), "sender", RPCErrorCode.INVALID_FIELDS)
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xa2f4134516fab4f38dfc2485820ac2b86352679f2af0e307c848adc3ceee43cf', id=682)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.eip7702.test_eip7702_tuple_userop" name="test_send_wrongchain_eip_7702_drop_userop" time="0.070"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32602, message='sender 0x95c0…9275 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=684)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/eip7702/test_eip7702_tuple_userop.py:244: in test_send_wrongchain_eip_7702_drop_userop
    assert_ok(userop.send())
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32602, message='sender 0x95c0…9275 is an existing contract, or the initCode 0x0000…0000 is not empty (but not both)', data=None, id=684)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.gas.test_eip_7623" name="test_normal_calldata" time="0.157"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32500, message='AA26 over verificationGasLimit', data=None, id=686)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/gas/test_eip_7623.py:35: in test_normal_calldata
    assert_ok(op.send())
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32500, message='AA26 over verificationGasLimit', data=None, id=686)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.gas.test_eip_7623" name="test_huge_calldata_failed" time="0.153"><failure message="AssertionError: assert -32500 == &lt;RPCErrorCode.INVALID_FIELDS: -32602&gt;&#10; +  where -32500 = Error(code=-32500, message='AA26 over verificationGasLimit', data=None, id=688).code">tests/single/gas/test_eip_7623.py:55: in test_huge_calldata_failed
    assert_rpc_error(ret, "preVerificationGas", RPCErrorCode.INVALID_FIELDS)
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AssertionError: assert -32500 == &lt;RPCErrorCode.INVALID_FIELDS: -32602&gt;
E    +  where -32500 = Error(code=-32500, message='AA26 over verificationGasLimit', data=None, id=688).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.gas.test_eip_7623" name="test_huge_calldata_with_callgas" time="0.183"><failure message="AssertionError: assert -32500 == &lt;RPCErrorCode.INVALID_FIELDS: -32602&gt;&#10; +  where -32500 = Error(code=-32500, message='AA26 over verificationGasLimit', data=None, id=690).code">tests/single/gas/test_eip_7623.py:74: in test_huge_calldata_with_callgas
    assert_rpc_error(ret, "preVerificationGas", RPCErrorCode.INVALID_FIELDS)
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AssertionError: assert -32500 == &lt;RPCErrorCode.INVALID_FIELDS: -32602&gt;
E    +  where -32500 = Error(code=-32500, message='AA26 over verificationGasLimit', data=None, id=690).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.gas.test_eip_7623" name="test_estimate_huge_calldata" time="10.024"><failure message="requests.exceptions.ReadTimeout: HTTPConnectionPool(host='localhost', port=3000): Read timed out. (read timeout=10)">.venv/lib/python3.10/site-packages/urllib3/connectionpool.py:536: in _make_request
    response = conn.getresponse()
.venv/lib/python3.10/site-packages/urllib3/connection.py:464: in getresponse
    httplib_response = super().getresponse()
../../../../.local/share/pdm/python/cpython@3.10.16/lib/python3.10/http/client.py:1375: in getresponse
    response.begin()
../../../../.local/share/pdm/python/cpython@3.10.16/lib/python3.10/http/client.py:318: in begin
    version, status, reason = self._read_status()
../../../../.local/share/pdm/python/cpython@3.10.16/lib/python3.10/http/client.py:279: in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
../../../../.local/share/pdm/python/cpython@3.10.16/lib/python3.10/socket.py:717: in readinto
    return self._sock.recv_into(b)
E   TimeoutError: timed out

The above exception was the direct cause of the following exception:
.venv/lib/python3.10/site-packages/requests/adapters.py:667: in send
    resp = conn.urlopen(
.venv/lib/python3.10/site-packages/urllib3/connectionpool.py:843: in urlopen
    retries = retries.increment(
.venv/lib/python3.10/site-packages/urllib3/util/retry.py:474: in increment
    raise reraise(type(error), error, _stacktrace)
.venv/lib/python3.10/site-packages/urllib3/util/util.py:39: in reraise
    raise value
.venv/lib/python3.10/site-packages/urllib3/connectionpool.py:789: in urlopen
    response = self._make_request(
.venv/lib/python3.10/site-packages/urllib3/connectionpool.py:538: in _make_request
    self._raise_timeout(err=e, url=url, timeout_value=read_timeout)
.venv/lib/python3.10/site-packages/urllib3/connectionpool.py:369: in _raise_timeout
    raise ReadTimeoutError(
E   urllib3.exceptions.ReadTimeoutError: HTTPConnectionPool(host='localhost', port=3000): Read timed out. (read timeout=10)

During handling of the above exception, another exception occurred:
tests/single/gas/test_eip_7623.py:94: in test_estimate_huge_calldata
    ).send()
tests/types.py:60: in send
    postres = requests.post(url, json=asdict(self), timeout=10).json()
.venv/lib/python3.10/site-packages/requests/api.py:115: in post
    return request("post", url, data=data, json=json, **kwargs)
.venv/lib/python3.10/site-packages/requests/api.py:59: in request
    return session.request(method=method, url=url, **kwargs)
.venv/lib/python3.10/site-packages/requests/sessions.py:589: in request
    resp = self.send(prep, **send_kwargs)
.venv/lib/python3.10/site-packages/requests/sessions.py:703: in send
    r = adapter.send(request, **kwargs)
.venv/lib/python3.10/site-packages/requests/adapters.py:713: in send
    raise ReadTimeout(e, request=request)
E   requests.exceptions.ReadTimeout: HTTPConnectionPool(host='localhost', port=3000): Read timed out. (read timeout=10)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_no_factory[CREATE]" time="0.245" /><testcase classname="tests.single.opbanning.test_create" name="test_account_no_factory[CREATE2]" time="0.253" /><testcase classname="tests.single.opbanning.test_create" name="test_paymaster_banned_create[CREATE]" time="0.419" /><testcase classname="tests.single.opbanning.test_create" name="test_paymaster_banned_create[CREATE2]" time="0.428" /><testcase classname="tests.single.opbanning.test_create" name="test_account_create2_once" time="0.424" /><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=unstaked,op=CREATE,expect=ok,rule=OP-032]" time="0.402"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=704)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_create.py:136: in test_account_create_with_factory
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=704)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=unstaked,op=CREATE2,expect=err,rule=OP-032]" time="0.417"><failure message="AssertionError: assert 'account uses banned opcode: create2' in 'factory uses banned opcode: create'&#10; +  where 'account uses banned opcode: create2' = &lt;built-in method lower of str object at 0x7f9b51c8b630&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f9b51c8b630&gt; = 'account uses banned opcode: CREATE2'.lower&#10; +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f9b51c8b270&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f9b51c8b270&gt; = 'factory uses banned opcode: CREATE'.lower&#10; +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=706).message">tests/single/opbanning/test_create.py:138: in test_account_create_with_factory
    assert_rpc_error(
tests/utils.py:207: in assert_rpc_error
    assert message.lower() in response.message.lower()
E   AssertionError: assert 'account uses banned opcode: create2' in 'factory uses banned opcode: create'
E    +  where 'account uses banned opcode: create2' = &lt;built-in method lower of str object at 0x7f9b51c8b630&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f9b51c8b630&gt; = 'account uses banned opcode: CREATE2'.lower
E    +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f9b51c8b270&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f9b51c8b270&gt; = 'factory uses banned opcode: CREATE'.lower
E    +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=706).message</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=staked,op=CREATE,expect=ok,rule=EREP-060]" time="0.346"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=708)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_create.py:136: in test_account_create_with_factory
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=708)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=staked,op=nested-CREATE,expect=err,rule=EREP-060]" time="0.541"><failure message="AssertionError: assert 'account uses banned opcode: create' in 'factory uses banned opcode: create'&#10; +  where 'account uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f9b519dbed0&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f9b519dbed0&gt; = 'account uses banned opcode: CREATE'.lower&#10; +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f9b51a5e670&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f9b51a5e670&gt; = 'factory uses banned opcode: CREATE'.lower&#10; +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=710).message">tests/single/opbanning/test_create.py:138: in test_account_create_with_factory
    assert_rpc_error(
tests/utils.py:207: in assert_rpc_error
    assert message.lower() in response.message.lower()
E   AssertionError: assert 'account uses banned opcode: create' in 'factory uses banned opcode: create'
E    +  where 'account uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f9b519dbed0&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f9b519dbed0&gt; = 'account uses banned opcode: CREATE'.lower
E    +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f9b51a5e670&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f9b51a5e670&gt; = 'factory uses banned opcode: CREATE'.lower
E    +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=710).message</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=staked,op=nested-CREATE2,expect=err,rule=EREP-060]" time="0.342"><failure message="AssertionError: assert 'account uses banned opcode: create2' in 'factory uses banned opcode: create'&#10; +  where 'account uses banned opcode: create2' = &lt;built-in method lower of str object at 0x7f9b51af4c30&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f9b51af4c30&gt; = 'account uses banned opcode: CREATE2'.lower&#10; +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f9b51a5d110&gt;()&#10; +    where &lt;built-in method lower of str object at 0x7f9b51a5d110&gt; = 'factory uses banned opcode: CREATE'.lower&#10; +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=712).message">tests/single/opbanning/test_create.py:138: in test_account_create_with_factory
    assert_rpc_error(
tests/utils.py:207: in assert_rpc_error
    assert message.lower() in response.message.lower()
E   AssertionError: assert 'account uses banned opcode: create2' in 'factory uses banned opcode: create'
E    +  where 'account uses banned opcode: create2' = &lt;built-in method lower of str object at 0x7f9b51af4c30&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f9b51af4c30&gt; = 'account uses banned opcode: CREATE2'.lower
E    +  and   'factory uses banned opcode: create' = &lt;built-in method lower of str object at 0x7f9b51a5d110&gt;()
E    +    where &lt;built-in method lower of str object at 0x7f9b51a5d110&gt; = 'factory uses banned opcode: CREATE'.lower
E    +      where 'factory uses banned opcode: CREATE' = Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=712).message</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_create" name="test_account_create_with_factory[factory=staked,op=CREATE2,expect=ok,rule=EREP-060]" time="0.360"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=714)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_create.py:136: in test_account_create_with_factory
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: CREATE', data=None, id=714)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GAS-]" time="0.335" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GAS-CALL:&gt;]" time="0.409" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GAS-DELEGATECALL:&gt;]" time="0.515" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[NUMBER-]" time="0.282" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[NUMBER-CALL:&gt;]" time="0.398" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[NUMBER-DELEGATECALL:&gt;]" time="0.564" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[TIMESTAMP-]" time="0.344" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[TIMESTAMP-CALL:&gt;]" time="0.547" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[TIMESTAMP-DELEGATECALL:&gt;]" time="0.428" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[COINBASE-]" time="0.348" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[COINBASE-CALL:&gt;]" time="0.625" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[COINBASE-DELEGATECALL:&gt;]" time="0.531" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[DIFFICULTY-]" time="0.278" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[DIFFICULTY-CALL:&gt;]" time="0.495" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[DIFFICULTY-DELEGATECALL:&gt;]" time="0.546" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BASEFEE-]" time="0.323" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BASEFEE-CALL:&gt;]" time="0.464" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BASEFEE-DELEGATECALL:&gt;]" time="0.531" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASLIMIT-]" time="0.341" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASLIMIT-CALL:&gt;]" time="0.394" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASLIMIT-DELEGATECALL:&gt;]" time="0.500" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASPRICE-]" time="0.217" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASPRICE-CALL:&gt;]" time="0.379" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASPRICE-DELEGATECALL:&gt;]" time="0.400" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFBALANCE-]" time="0.241" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFBALANCE-CALL:&gt;]" time="0.509" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFBALANCE-DELEGATECALL:&gt;]" time="0.527" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BALANCE-]" time="0.320" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BALANCE-CALL:&gt;]" time="0.373" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BALANCE-DELEGATECALL:&gt;]" time="0.502" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[ORIGIN-]" time="0.220" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[ORIGIN-CALL:&gt;]" time="0.376" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[ORIGIN-DELEGATECALL:&gt;]" time="0.497" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOCKHASH-]" time="0.223" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOCKHASH-CALL:&gt;]" time="0.387" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOCKHASH-DELEGATECALL:&gt;]" time="0.408" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE-]" time="0.336" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE-CALL:&gt;]" time="0.487" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE-DELEGATECALL:&gt;]" time="0.513" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE2-]" time="0.340" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE2-CALL:&gt;]" time="0.490" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE2-DELEGATECALL:&gt;]" time="0.415" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFDESTRUCT-]" time="0.338" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFDESTRUCT-CALL:&gt;]" time="0.509" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFDESTRUCT-DELEGATECALL:&gt;]" time="0.423" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBHASH-]" time="0.369"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x38c34987cccbdb6183887e6c872e46e0b3335d57d165692138d484706dbad431', id=806)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x38c34987cccbdb6183887e6c872e46e0b3335d57d165692138d484706dbad431', id=806)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBHASH-CALL:&gt;]" time="0.495"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x7c05dfe20073561fa5d1ddad900c642ec0244bdb8f7cf6d4a568489b0742a286', id=808)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x7c05dfe20073561fa5d1ddad900c642ec0244bdb8f7cf6d4a568489b0742a286', id=808)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBHASH-DELEGATECALL:&gt;]" time="0.412"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xf5111bb4af44f6ea1a4b45d854e559290ec1113bbf608a017abf2aa0cb0a28bf', id=810)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xf5111bb4af44f6ea1a4b45d854e559290ec1113bbf608a017abf2aa0cb0a28bf', id=810)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBBASEFEE-]" time="0.234"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x1e434d154f9873a18a4b12dd9afb62a8396e9fb4ae2ac01a5eb9d04da348e239', id=812)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x1e434d154f9873a18a4b12dd9afb62a8396e9fb4ae2ac01a5eb9d04da348e239', id=812)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBBASEFEE-CALL:&gt;]" time="0.500"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xde1e324231903c010fd06ac6aaccdc052798f68a7391bfc8e1d1099daac82816', id=814)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xde1e324231903c010fd06ac6aaccdc052798f68a7391bfc8e1d1099daac82816', id=814)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOBBASEFEE-DELEGATECALL:&gt;]" time="0.419"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xe6663eb8b32f5fd56e3bb7e23db26559e4865e80480c0b64e63b124079e11742', id=816)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:55: in test_account_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xe6663eb8b32f5fd56e3bb7e23db26559e4865e80480c0b64e63b124079e11742', id=816)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[BALANCE-]" time="0.342"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=818)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=818)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[BALANCE-CALL:&gt;]" time="0.497"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=820)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=820)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[BALANCE-DELEGATECALL:&gt;]" time="0.520"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=822)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: BALANCE', data=None, id=822)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[SELFBALANCE-]" time="0.261"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=824)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=824)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[SELFBALANCE-CALL:&gt;]" time="0.536"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=826)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=826)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_when_staked[SELFBALANCE-DELEGATECALL:&gt;]" time="0.551"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=828)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:69: in test_account_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='account uses banned opcode: SELFBALANCE', data=None, id=828)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS CALL-]" time="0.363" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS CALL-CALL:&gt;]" time="0.427" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS CALL-DELEGATECALL:&gt;]" time="0.438" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS DELEGATECALL-]" time="0.264" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS DELEGATECALL-CALL:&gt;]" time="0.473" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS DELEGATECALL-DELEGATECALL:&gt;]" time="0.458" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GAS-]" time="0.368" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GAS-CALL:&gt;]" time="0.535" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GAS-DELEGATECALL:&gt;]" time="0.627" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[NUMBER-]" time="0.367" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[NUMBER-CALL:&gt;]" time="0.577" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[NUMBER-DELEGATECALL:&gt;]" time="0.665" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[TIMESTAMP-]" time="0.391" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[TIMESTAMP-CALL:&gt;]" time="0.513" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[TIMESTAMP-DELEGATECALL:&gt;]" time="0.595" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[COINBASE-]" time="0.291" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[COINBASE-CALL:&gt;]" time="0.597" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[COINBASE-DELEGATECALL:&gt;]" time="0.691" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[DIFFICULTY-]" time="0.407" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[DIFFICULTY-CALL:&gt;]" time="0.631" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[DIFFICULTY-DELEGATECALL:&gt;]" time="0.610" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BASEFEE-]" time="0.282" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BASEFEE-CALL:&gt;]" time="0.591" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BASEFEE-DELEGATECALL:&gt;]" time="0.576" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASLIMIT-]" time="0.394" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASLIMIT-CALL:&gt;]" time="0.604" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASLIMIT-DELEGATECALL:&gt;]" time="0.589" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASPRICE-]" time="0.398" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASPRICE-CALL:&gt;]" time="0.505" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASPRICE-DELEGATECALL:&gt;]" time="0.693" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFBALANCE-]" time="0.470" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFBALANCE-CALL:&gt;]" time="0.674" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFBALANCE-DELEGATECALL:&gt;]" time="0.649" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BALANCE-]" time="0.389" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BALANCE-CALL:&gt;]" time="0.590" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BALANCE-DELEGATECALL:&gt;]" time="0.681" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[ORIGIN-]" time="0.382" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[ORIGIN-CALL:&gt;]" time="0.485" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[ORIGIN-DELEGATECALL:&gt;]" time="0.677" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOCKHASH-]" time="0.414" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOCKHASH-CALL:&gt;]" time="0.619" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOCKHASH-DELEGATECALL:&gt;]" time="0.711" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE-]" time="0.393" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE-CALL:&gt;]" time="0.603" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE-DELEGATECALL:&gt;]" time="0.690" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE2-]" time="0.303" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE2-CALL:&gt;]" time="0.613" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE2-DELEGATECALL:&gt;]" time="0.701" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFDESTRUCT-]" time="0.450" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFDESTRUCT-CALL:&gt;]" time="0.671" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFDESTRUCT-DELEGATECALL:&gt;]" time="0.651" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBHASH-]" time="0.413"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xd8fecc6d4d2c18175579ac717f42ef2fc984517e49a6b9f74fc02e462c726194', id=932)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xd8fecc6d4d2c18175579ac717f42ef2fc984517e49a6b9f74fc02e462c726194', id=932)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBHASH-CALL:&gt;]" time="0.513"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xecf749a1d58586e6fbde8be376a96ef4f30c86c23a1ef29c525e0d2e3d20f301', id=934)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xecf749a1d58586e6fbde8be376a96ef4f30c86c23a1ef29c525e0d2e3d20f301', id=934)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBHASH-DELEGATECALL:&gt;]" time="0.709"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x84d7f04171282e0ef9990c9dcba2cc90331f47b2b1cca3638ac25682635e6ca0', id=936)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x84d7f04171282e0ef9990c9dcba2cc90331f47b2b1cca3638ac25682635e6ca0', id=936)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBBASEFEE-]" time="0.472"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x6f3dd74df4c4767bf624517a755219194fa10b121bd8c836eb804d63427d27f5', id=938)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x6f3dd74df4c4767bf624517a755219194fa10b121bd8c836eb804d63427d27f5', id=938)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBBASEFEE-CALL:&gt;]" time="0.548"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x25f36adeb3113810a07513db729c5ff6f5dd417d4df6e0f83e49c24ca285af8f', id=940)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x25f36adeb3113810a07513db729c5ff6f5dd417d4df6e0f83e49c24ca285af8f', id=940)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOBBASEFEE-DELEGATECALL:&gt;]" time="0.737"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x106fe10aa601d0a30e09a871ca082643f2b4410c00c735f325e091fe0ab4952d', id=942)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:96: in test_paymaster_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x106fe10aa601d0a30e09a871ca082643f2b4410c00c735f325e091fe0ab4952d', id=942)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_allowed_opcode_when_staked[BALANCE]" time="0.418"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='paymaster uses banned opcode: BALANCE', data=None, id=944)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:113: in test_paymaster_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='paymaster uses banned opcode: BALANCE', data=None, id=944)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_allowed_opcode_when_staked[SELFBALANCE]" time="0.468"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='paymaster uses banned opcode: SELFBALANCE', data=None, id=946)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:113: in test_paymaster_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='paymaster uses banned opcode: SELFBALANCE', data=None, id=946)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[GAS]" time="0.357" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[NUMBER]" time="0.255" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[TIMESTAMP]" time="0.360" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[COINBASE]" time="0.362" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[DIFFICULTY]" time="0.365" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BASEFEE]" time="0.367" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[GASLIMIT]" time="0.365" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[GASPRICE]" time="0.372" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[SELFBALANCE]" time="0.391" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BALANCE]" time="0.370" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[ORIGIN]" time="0.370" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BLOCKHASH]" time="0.272" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[CREATE]" time="0.382" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[CREATE2]" time="0.382" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[SELFDESTRUCT]" time="0.386" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BLOBHASH]" time="0.281"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0x877a513ba16bd2e6464065ceba37aa94af17d5157dd6895db27f6297f27588f3', id=978)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:127: in test_factory_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0x877a513ba16bd2e6464065ceba37aa94af17d5157dd6895db27f6297f27588f3', id=978)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BLOBBASEFEE]" time="0.383"><failure message="AttributeError: expected error object, got:&#10;Ok(result='0xdc7df70dcdb2f7ec342d4e649835e0585268484e42d06766c7e6fde8c9866803', id=980)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:127: in test_factory_banned_opcode
    assert_rpc_error(
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result='0xdc7df70dcdb2f7ec342d4e649835e0585268484e42d06766c7e6fde8c9866803', id=980)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_allowed_opcode_when_staked[BALANCE]" time="0.393"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: BALANCE', data=None, id=982)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:153: in test_factory_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: BALANCE', data=None, id=982)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_allowed_opcode_when_staked[SELFBALANCE]" time="0.415"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32502, message='factory uses banned opcode: SELFBALANCE', data=None, id=984)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/opbanning/test_op_banning.py:153: in test_factory_allowed_opcode_when_staked
    assert_ok(response)
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32502, message='factory uses banned opcode: SELFBALANCE', data=None, id=984)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_erep" name="test_paymaster_on_account_failure" time="0.148"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/reputation/test_erep.py:71: in test_paymaster_on_account_failure
    set_reputation(paymaster.address, ops_seen=5, ops_included=2)
tests/utils.py:321: in set_reputation
    assert res.result
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_erep" name="test_staked_factory_on_account_failure" time="0.836"><failure message="AttributeError: expected result object, got:&#10;Error(code=-32603, message='gRPC error: Send bundle now with error: (code: 3, message: execution reverted, data: Some(String(&quot;0x65c8fd4d0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000000d4141323320726576657274656400000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000006408c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000114e6f2062756e646c6520666f7220796f7500000000000000000000000000000000000000000000000000000000000000000000000000000000000000&quot;)))\n\nLocation:\n    /app/crates/bundler/src/bundler.rs:123:29', data=None, id=994)">tests/utils.py:199: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/reputation/test_erep.py:119: in test_staked_factory_on_account_failure
    send_bundle_now()
tests/utils.py:230: in send_bundle_now
    assert_ok(RPCRequest(method="debug_bundler_sendBundleNow").send(url))
tests/utils.py:201: in assert_ok
    raise AttributeError(f"expected result object, got:\n{response}") from exc
E   AttributeError: expected result object, got:
E   Error(code=-32603, message='gRPC error: Send bundle now with error: (code: 3, message: execution reverted, data: Some(String("0x65c8fd4d0000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000000d4141323320726576657274656400000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000006408c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000114e6f2062756e646c6520666f7220796f7500000000000000000000000000000000000000000000000000000000000000000000000000000000000000")))\n\nLocation:\n    /app/crates/bundler/src/bundler.rs:123:29', data=None, id=994)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_erep" name="test_account_on_entity_failure[staked]" time="0.378"><failure message="AssertionError: staked account should be blamed instead of paymaster&#10;assert [Reputation(a...=0, status=0)] == [Reputation(a...=0, status=0)]&#10;  &#10;  At index 1 diff: Reputation(address='0xf66db0bede6f1d510749018b145d574df91cadc2', opsSeen=1, opsIncluded=0, status=0) != Reputation(address='0xf66db0bede6f1d510749018b145d574df91cadc2', opsSeen=0, opsIncluded=0, status=0)&#10;  &#10;  Full diff:&#10;    [&#10;        Reputation(&#10;            address='0x2e8efd72bba6a9c4f57775392525a1d2054f082c',&#10;            opsSeen=1,&#10;            opsIncluded=0,&#10;            status=0,&#10;        ),&#10;        Reputation(&#10;            address='0xf66db0bede6f1d510749018b145d574df91cadc2',&#10;  -         opsSeen=0,&#10;  ?                 ^&#10;  +         opsSeen=1,&#10;  ?                 ^&#10;            opsIncluded=0,&#10;            status=0,&#10;        ),&#10;    ]">tests/single/reputation/test_erep.py:174: in test_account_on_entity_failure
    assert get_reputations([sender.address, paymaster.address]) == [
E   AssertionError: staked account should be blamed instead of paymaster
E   assert [Reputation(a...=0, status=0)] == [Reputation(a...=0, status=0)]
E     
E     At index 1 diff: Reputation(address='0xf66db0bede6f1d510749018b145d574df91cadc2', opsSeen=1, opsIncluded=0, status=0) != Reputation(address='0xf66db0bede6f1d510749018b145d574df91cadc2', opsSeen=0, opsIncluded=0, status=0)
E     
E     Full diff:
E       [
E           Reputation(
E               address='0x2e8efd72bba6a9c4f57775392525a1d2054f082c',
E               opsSeen=1,
E               opsIncluded=0,
E               status=0,
E           ),
E           Reputation(
E               address='0xf66db0bede6f1d510749018b145d574df91cadc2',
E     -         opsSeen=0,
E     ?                 ^
E     +         opsSeen=1,
E     ?                 ^
E               opsIncluded=0,
E               status=0,
E           ),
E       ]</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------
staking account &lt;web3._utils.datatypes.Contract object at 0x7f9b51ea33d0&gt;

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_erep" name="test_account_on_entity_failure[unstaked]" time="0.273"><failure message="AssertionError: valid userop. staked paymaster should have opsSeen increment&#10;assert [Reputation(a...=0, status=0)] == [Reputation(a...=0, status=0)]&#10;  &#10;  At index 0 diff: Reputation(address='0x99fbe216531bce23bac9529c4b8cb2add1bcfa8b', opsSeen=1, opsIncluded=0, status=0) != Reputation(address='0x99fbe216531bce23bac9529c4b8cb2add1bcfa8b', opsSeen=0, opsIncluded=0, status=0)&#10;  &#10;  Full diff:&#10;    [&#10;        Reputation(&#10;            address='0x99fbe216531bce23bac9529c4b8cb2add1bcfa8b',&#10;  -         opsSeen=0,&#10;  ?                 ^&#10;  +         opsSeen=1,&#10;  ?                 ^&#10;            opsIncluded=0,&#10;            status=0,&#10;        ),&#10;        Reputation(&#10;            address='0x5efb11eec9dc89b6870720ca2450a20cef4a690e',&#10;            opsSeen=1,&#10;            opsIncluded=0,&#10;            status=0,&#10;        ),&#10;    ]">tests/single/reputation/test_erep.py:156: in test_account_on_entity_failure
    assert get_reputations([sender.address, paymaster.address]) == [
E   AssertionError: valid userop. staked paymaster should have opsSeen increment
E   assert [Reputation(a...=0, status=0)] == [Reputation(a...=0, status=0)]
E     
E     At index 0 diff: Reputation(address='0x99fbe216531bce23bac9529c4b8cb2add1bcfa8b', opsSeen=1, opsIncluded=0, status=0) != Reputation(address='0x99fbe216531bce23bac9529c4b8cb2add1bcfa8b', opsSeen=0, opsIncluded=0, status=0)
E     
E     Full diff:
E       [
E           Reputation(
E               address='0x99fbe216531bce23bac9529c4b8cb2add1bcfa8b',
E     -         opsSeen=0,
E     ?                 ^
E     +         opsSeen=1,
E     ?                 ^
E               opsIncluded=0,
E               status=0,
E           ),
E           Reputation(
E               address='0x5efb11eec9dc89b6870720ca2450a20cef4a690e',
E               opsSeen=1,
E               opsIncluded=0,
E               status=0,
E           ),
E       ]</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------
unstaked account &lt;web3._utils.datatypes.Contract object at 0x7f9b51fddf30&gt;

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_replace" name="test_replace_paymaster" time="0.567"><failure message="AssertionError: replace with unmodified userop&#10;assert 2 == 1&#10; +  where 2 = pm_opsSeen('0x8f518bE301816b52A5933A4ac2Fbd1Ea12FD69DB')&#10; +    where '0x8f518bE301816b52A5933A4ac2Fbd1Ea12FD69DB' = &lt;web3._utils.datatypes.Contract object at 0x7f9b51ab1a80&gt;.address">tests/single/reputation/test_replace.py:37: in test_replace_paymaster
    assert pm_opsSeen(pm1.address) == 1, "replace with unmodified userop"
E   AssertionError: replace with unmodified userop
E   assert 2 == 1
E    +  where 2 = pm_opsSeen('0x8f518bE301816b52A5933A4ac2Fbd1Ea12FD69DB')
E    +    where '0x8f518bE301816b52A5933A4ac2Fbd1Ea12FD69DB' = &lt;web3._utils.datatypes.Contract object at 0x7f9b51ab1a80&gt;.address</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_reputation" name="test_staked_entity_reputation_threshold[with_factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/reputation/test_reputation.py:61: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_reputation" name="test_staked_entity_reputation_threshold[without_factory]" time="0.000"><skipped type="pytest.skip" message="skipped">/home/runner/work/bundler-test-executor/bundler-test-executor/bundler-spec-tests/tests/single/reputation/test_reputation.py:61: skipped</skipped><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_chainId" name="test_eth_chainId[]" time="0.008" /><testcase classname="tests.single.rpc.test_eth_estimateUserOperationGas" name="test_eth_estimateUserOperationGas[]" time="0.169" /><testcase classname="tests.single.rpc.test_eth_estimateUserOperationGas" name="test_eth_estimateUserOperationGas_execution_revert" time="0.183"><failure message="AttributeError: expected error object, got:&#10;Ok(result={'preVerificationGas': '0xb413', 'verificationGasLimit': '0xeffe', 'callGasLimit': '0x2f44'}, id=1022)">tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/rpc/test_eth_estimateUserOperationGas.py:32: in test_eth_estimateUserOperationGas_execution_revert
    assert_rpc_error(response, "", RPCErrorCode.EXECUTION_REVERTED)
tests/utils.py:211: in assert_rpc_error
    raise AttributeError(f"expected error object, got:\n{response}") from exc
E   AttributeError: expected error object, got:
E   Ok(result={'preVerificationGas': '0xb413', 'verificationGasLimit': '0xeffe', 'callGasLimit': '0x2f44'}, id=1022)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_estimateUserOperationGas" name="test_eth_estimateUserOperationGas_simulation_revert" time="0.029"><failure message="assert -32603 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;&#10; +  where -32603 = Error(code=-32603, message='other error: FailedOpWithRevert(FailedOpWithRevert { op_index: 0, reason: &quot;AA23 reverted&quot;,...0000000000000000000000000001a7465737457616c6c65743a2064656164207369676e6174757265000000000000) })', data=None, id=1024).code">tests/single/rpc/test_eth_estimateUserOperationGas.py:40: in test_eth_estimateUserOperationGas_simulation_revert
    assert_rpc_error(response, "dead signature", RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT)
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   assert -32603 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;
E    +  where -32603 = Error(code=-32603, message='other error: FailedOpWithRevert(FailedOpWithRevert { op_index: 0, reason: "AA23 reverted",...0000000000000000000000000001a7465737457616c6c65743a2064656164207369676e6174757265000000000000) })', data=None, id=1024).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_getUserOperationByHash" name="test_eth_getUserOperationByHash[]" time="0.354"><failure message="jsonschema.exceptions.ValidationError: None is not valid under any of the given schemas&#10;&#10;Failed validating 'oneOf' in schema['properties']['userOperation']['properties']['paymasterVerificationGasLimit']:&#10;    {'title': 'paymaster verification gas limit',&#10;     'oneOf': [{'type': 'string', 'pattern': '^0x([1-9a-f]+[0-9a-f]*|0)$'},&#10;               {'type': 'number'}],&#10;     'description': 'The amount of gas to allocate for the paymaster '&#10;                    'validation code'}&#10;&#10;On instance['userOperation']['paymasterVerificationGasLimit']:&#10;    None">tests/single/rpc/test_eth_getUserOperationByHash.py:24: in test_eth_getUserOperationByHash
    validate(instance=response.result, schema=schema)
.venv/lib/python3.10/site-packages/jsonschema/validators.py:1332: in validate
    raise error
E   jsonschema.exceptions.ValidationError: None is not valid under any of the given schemas
E   
E   Failed validating 'oneOf' in schema['properties']['userOperation']['properties']['paymasterVerificationGasLimit']:
E       {'title': 'paymaster verification gas limit',
E        'oneOf': [{'type': 'string', 'pattern': '^0x([1-9a-f]+[0-9a-f]*|0)$'},
E                  {'type': 'number'}],
E        'description': 'The amount of gas to allocate for the paymaster '
E                       'validation code'}
E   
E   On instance['userOperation']['paymasterVerificationGasLimit']:
E       None</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_getUserOperationByHash" name="test_eth_getUserOperationByHash_error" time="0.005" /><testcase classname="tests.single.rpc.test_eth_getUserOperationReceipt" name="test_eth_getUserOperationReceipt[]" time="0.368" /><testcase classname="tests.single.rpc.test_eth_getUserOperationReceipt" name="test_eth_getUserOperationReceipt_error" time="0.004" /><testcase classname="tests.single.rpc.test_eth_sendUserOperation" name="test_eth_sendUserOperation[]" time="0.331"><failure message="AssertionError: assert '0x0ad5d5c352...67fc3e69f4ffd' == '0x5f441e9a58...1dc5963730c3c'&#10;  &#10;  - 0x5f441e9a580daee4650b1cacea96fe798ac8fc947adf8312ee41dc5963730c3c&#10;  + 0x0ad5d5c352be9cec637ac8f8311570ea62527827dfacf3ec10467fc3e69f4ffd">tests/single/rpc/test_eth_sendUserOperation.py:19: in test_eth_sendUserOperation
    assert response.result == userop_hash(helper_contract, userop)
E   AssertionError: assert '0x0ad5d5c352...67fc3e69f4ffd' == '0x5f441e9a58...1dc5963730c3c'
E     
E     - 0x5f441e9a580daee4650b1cacea96fe798ac8fc947adf8312ee41dc5963730c3c
E     + 0x0ad5d5c352be9cec637ac8f8311570ea62527827dfacf3ec10467fc3e69f4ffd</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_sendUserOperation" name="test_eth_sendUserOperation_revert" time="0.043"><failure message="AssertionError: assert -32603 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;&#10; +  where -32603 = Error(code=-32603, message='other error: 0, AA23 reverted, 0x08c379a00000000000000000000000000000000000000000000000000...00000000000000000000000000000001a7465737457616c6c65743a2064656164207369676e6174757265000000000000', data=None, id=1041).code">tests/single/rpc/test_eth_sendUserOperation.py:32: in test_eth_sendUserOperation_revert
    assert_rpc_error(
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AssertionError: assert -32603 == &lt;RPCErrorCode.REJECTED_BY_EP_OR_ACCOUNT: -32500&gt;
E    +  where -32603 = Error(code=-32603, message='other error: 0, AA23 reverted, 0x08c379a00000000000000000000000000000000000000000000000000...00000000000000000000000000000001a7465737457616c6c65743a2064656164207369676e6174757265000000000000', data=None, id=1041).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_sendUserOperation" name="test_eth_sendUserOperation_invalid_signature" time="0.028"><failure message="AssertionError: assert -32503 == &lt;RPCErrorCode.INVALID_SIGNATURE: -32507&gt;&#10; +  where -32503 = Error(code=-32503, message='already expired', data=None, id=1044).code">tests/single/rpc/test_eth_sendUserOperation.py:39: in test_eth_sendUserOperation_invalid_signature
    assert_rpc_error(
tests/utils.py:206: in assert_rpc_error
    assert response.code == code
E   AssertionError: assert -32503 == &lt;RPCErrorCode.INVALID_SIGNATURE: -32507&gt;
E    +  where -32503 = Error(code=-32503, message='already expired', data=None, id=1044).code</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_supportedEntryPoints" name="test_eth_supportedEntryPoints[]" time="0.009" /></testsuite></testsuites>